cmd/clusterctl/client/cluster/mover.go

```diff
@@ -200,8 +211,10 @@ func (o *objectMover) move(graph *objectGraph, toProxy Proxy) error {
 
 	// Ensure all the expected target namespaces are in place before creating objects.
 	log.V(1).Info("Creating target namespaces, if missing")
-	if err := o.ensureNamespaces(graph, toProxy); err != nil {
-		return err
+	if o.isDryrun {
```

Comment:
Surely we only want to do this if this isn't dry run? Think this might be a mistake

Reply:
@JoelSpeed I am still working on it to clean up some stuffs , I updated the PR title with the issue and 
yes, the purpose is to only list resource to be moved but not actually take any actions..